basic/stam/stam_exercise.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import os
+
+PATH = 1
+
+
+def open_file(directory, mode):
+    input_file = open(directory, mode)
+    return input_file
+
+
+def main():
+    try:
+        directory = sys.argv[PATH]
+        if ".txt" in directory:
+            pass
+        else:
+            print(os.listdir(directory))
+            file_name = input("Enter file name from list above: ")
+            directory += "\\" + file_name
```

Comment:
Nice Feature but not needed, you can safely assume you get a valid file name

Reply:
O.k. I think it's fine to leave it. If you would like I will ommit it.